docker/start.sh

```diff
@@ -15,9 +15,9 @@ else
   echo "{\"courses\":[\"${OCW_TEST_COURSE}\"]}" | tee /ocw-data/courses.json
   cd /ocw-to-hugo && node . -i ${OCW_TO_HUGO_INPUT:-/ocw-data/input} \
     -o /ocw-data/output -c /ocw-data/courses.json --download ${OCW_TO_HUGO_DOWNLOAD:-true}
-  rm -rf /ocw-to-hugo-output/*
+  rm -rf /ocw-to-hugo-output/* && mkdir -p /ocw-to-hugo-output/config/_default
```

Comment:
Can we split these onto separate lines? The `&&` doesn't do anything useful here since the `set -e -o pipefail` will cause the script to stop if `rm` returns a `non-zero` exit code

Reply:
I separated those two lines.  Are there others you think we should separate here?  Is the general idea to only chain commands you want to be dependent on each other, so if anything "upstream" fails the program will exit before getting to the dependent code?

Reply:
I think we should break up all lines with `&&` in this file but I have another PR in this repo so I can do that there. Yes, `set -e -o pipefail` causes the script to stop when any command (including piped commands) returns a non-zero exit code. That's sometimes not desirable for complicated bash scripts because the exit codes may be there for control flow or other reasons besides error handling. (Although you can `set +e` temporarily and then `set -e` after to change behavior for a block of code.) `set -e -o pipefail` will also return a non-zero exit code for this script itself which is very useful because otherwise errors can be missed.